.github/workflows/conformance-clustermesh.yaml

```diff
@@ -0,0 +1,504 @@
+name: ClusterMesh
```

Comment:
It would be nice to have trigger phrases written in here like other workflows.

https://github.com/cilium/cilium/blob/master/.github/workflows/conformance-multicluster.yaml#L1